src/d3d11/d3d11-query.cpp

```
@@ -29,34 +29,128 @@ Result QueryPoolImpl::init()
     DeviceImpl* device = getDevice<DeviceImpl>();
-    if (!m_queries[index])
-        device->m_device->CreateQuery(&m_queryDesc, m_queries[index].writeRef());
-    return m_queries[index].get();
+    if (!m_queries[index].timestampQuery)
+        device->m_device->CreateQuery(&m_queryDesc, m_queries[index].timestampQuery.writeRef());
+    return m_queries[index].timestampQuery.get();
 }
 
-Result QueryPoolImpl::getResult(uint32_t queryIndex, uint32_t count, uint64_t* outData)
+void QueryPoolImpl::setDisjointQuery(uint32_t index, ID3D11Query* disjointQuery)
 {
```

Cppcheck (2.20.0)

[style] The function 'push' is never used.

(unusedFunction)